portal-api/src/scheduler/jobs/send-report-due-emails-job/index.js

```diff
@@ -0,0 +1,113 @@
+const { format, isSameDay } = require('date-fns');
+const { getReportDueDate, getReportPeriodMonthAndYear, getFormattedReportPeriod } = require('../helpers/utilisation-report-helpers');
+const api = require('../../../v1/api');
+const { hasValue, isValidEmail } = require('../../../utils/string');
+const sendEmail = require('../../../external-api/send-email');
+const EMAIL_TEMPLATE_IDS = require('../../../constants/email-template-ids');
+
+const { PORTAL_REPORT_DUE_EMAIL_SCHEDULE } = process.env;
+
+const EMAIL_NAME = 'GEF utilisation report due';
+
+/**
+ * Sends the email to the specified bank when a valid payment officer team email is present
+ * @param bank {object} - the bank to send the email to
+ * @param reportPeriod {string} - the formatted reporting period that will appear in the email
+ */
+const sendEmailForBank = async (bank, reportPeriod) => {
+  const { name: bankName, paymentOfficerTeam } = bank;
+  const paymentOfficerTeamEmail = paymentOfficerTeam?.email;
+
+  try {
+    if (!hasValue(paymentOfficerTeamEmail)) {
+      console.warn(`Not sending ${EMAIL_NAME} email to '${bankName}' - no payment officer team email set`);
+    } else if (!isValidEmail(paymentOfficerTeamEmail)) {
+      console.error(`Failed to send ${EMAIL_NAME} email to '${bankName}' - invalid payment officer team email '${paymentOfficerTeamEmail}'`);
+    } else {
+      await sendEmail(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_DUE_TODAY, paymentOfficerTeamEmail, {
+        recipient: paymentOfficerTeam.teamName,
```

Comment:
We'll want to send the email even if we don't have a teamName, but just because we won't really look at what emails are sent we'll not know if teamName isn't populated unless a bank complains.
Can we add a simple console.warn on a check if teamName is null somewhere?